src/integer/mat_poly_over_z/transpose.rs

```diff
@@ -0,0 +1,69 @@
+// Copyright © 2023 Marcel Luca Schmidt
+//
+// This file is part of qFALL-math.
+//
+// qFALL-math is free software: you can redistribute it and/or modify it under
+// the terms of the Mozilla Public License Version 2.0 as published by the
+// Mozilla Foundation. See <https://mozilla.org/en-US/MPL/2.0/>.
+
+//! This module contains the implementation of the `transpose` function.
+
+use crate::traits::*;
+
+use super::MatPolyOverZ;
+use flint_sys::fmpz_poly_mat::fmpz_poly_mat_transpose;
```

Comment:
```suggestion
use super::MatPolyOverZ;
use crate::traits::{GetNumColumns, GetNumRows};
use flint_sys::fmpz_poly_mat::fmpz_poly_mat_transpose;
```
Please specify the used traits. Furthermore, remove the newline. The suggestion should satisfy rustfmt and these two requirements.